pyproject.toml

```diff
@@ -1,3 +1,61 @@
 [build-system]
-requires = ["setuptools >= 56.2.0", "gitpython >= 3.1.18", "wheel"]
-build-backend = "setuptools.build_meta:__legacy__"
\ No newline at end of file
+requires = ["setuptools>=61",  "gitpython >= 3.1.18", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "picca"
+dynamic = ["dependencies", "version", "readme"]
+description = "Package for Igm Cosmological-Correlations Analyses"
+requires-python = ">=3.7"
+license = "GPL-3.0-or-later"
+license-files = ["LICENSE" ]
+authors = [
+    { name = "Nicolas Busca" },
+    { name = "Helion du Mas des Bourboux" },
+    { name = "Ignasi Pérez-Ràfols", email = "iprafols@gmail.com" },
+    { name = "Michael Walther" },
```

Comment:
this could be extended by more collaborators, the Email should probably be the one of a current TG lead

Reply:
This PR is not responsible for the email list, but only for moving setup.py to pyproject.toml